wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -916,6 +919,38 @@ abstract class Wallet
       this
     }
   }
+
+  def startFeeRateCallbackScheduler(): Unit = {
+    val feeRateChangedRunnable = new Runnable {
+      private val lastFeeRate =
+        new Mutable[Try[FeeUnit]](Failure(new RuntimeException("init")))
+      override def run(): Unit = {
+        getFeeRate()
+          .map(feeRate => Success(feeRate))
+          .recover { case NonFatal(ex) =>
+            logger.error("Cannot get fee rate ", ex)
+            Failure(ex)
+          }
+          .foreach { tryFeeRate =>
+            lastFeeRate.atomicUpdate(tryFeeRate) { (oldRate, newRate) =>
+              if (oldRate != newRate) {
```

Comment:
Do we not want to throw it all the time just to be safe? The frontend may not have been around to receive it previously.